tests/acceptance/features/webUIFiles/breadcrumb.feature

```diff
@@ -0,0 +1,26 @@
+Feature: access breadcrumb
+  As a user
+  I want to browse to parent folders using breadcrumb
+  So that I can access resources with ease
+
+  Background:
+    Given user "user1" has been created with default attributes
+
+  @issue-2538
+  Scenario: Check breadCrumb for folder one level below the root folder when rootFolder is set
+    Given rootFolder has been set to "" in config.json
+    And user "user1" has logged in using the webUI
+    When the user opens folder "simple-folder" using the webUI
+    #    Then breadcrumb for folder "simple-folder" should be displayed on the webUI
+    Then there should be no breadcrumb displayed on the webUI
+    When the user opens folder "simple-empty-folder" using the webUI
+    Then breadcrumb for folder "simple-empty-folder" should be displayed on the webUI
+
+  Scenario: Change rootFolder to simple-folder and check for the displayed files
+    Given rootFolder has been set to "simple-folder" in config.json
+    And user "user1" has logged in using the webUI
+    When the user browses to the files page
+    Then folder "0" should not be listed on the webUI
+    But as "user1" folder "0" should exist
+    And file "lorem.txt" should be listed on the webUI
+    And the content of file "simple-folder/lorem.txt" for user "user1" with range "0-39" bytes should be "This is lorem text in the simple-folder"
```

Comment:
I don't think we need that check here, the config change will only affect phoenix but that check is done by webDAV